src/App.js

```diff
@@ -1,19 +1,49 @@
 import React from 'react';
 import './App.css';
 import chatMessages from './data/messages.json';
+import ChatLog from './components/ChatLog';
+import { useState } from 'react';
+
 
 const App = () => {
+  const [chatData, setChatData] = useState(chatMessages)
+  const [totalLikes, setTotalLikes] = useState(0)
```

Comment:
Great use of state here! One thing to think about is whether or not we need the state for totalLikes. The total likes can be stored as a local variable and calculated pretty easily below!